src/neo4j_graphrag/embeddings/vertexai.py

```diff
@@ -38,7 +35,14 @@ class VertexAIEmbeddings(Embedder):
     """
 
     def __init__(self, model: str = "text-embedding-004") -> None:
-        self.vertexai_model = TextEmbeddingModel.from_pretrained(model)
+        if vertexai is None:
```

Comment:
Have fixed this in my PR if you're still having issues

Reply:
Ok, will rebase after your PR is merged, thanks